tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -46,7 +46,7 @@ raw_amp = <span>foo & bar</span>
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 many_nbsp = <div>&nbsp; &nbsp; </div>;
 single_nbsp = <div>&nbsp;</div>;
-many_raw_nbsp = <div>   </div>;
+many_raw_nbsp = <div>   </div>;
```

Comment:
These have turned from regular spaces back to non-breaking spaces, right? That's great!